packages/protocol/contracts/common/Accounts.sol

```diff
@@ -597,14 +629,24 @@ contract Accounts is
   }
 
   /**
-   * @notice Returns the signer for the specified account and role. If no 
-   * signer has been specified it will return the account itself.
+   * @notice Checks whether the role is one of Vote, Validator or 
+   * Attestation
+   * @param role The role to check
+   */
+  function isLegacyRole(bytes32 role) public view returns (bool) {
+    return role == ValidatorSigner || role == AttestationSigner || role == VoteSigner;
```

Comment:
this is slightly pedantic but I would order these with `VoteSigner` first for more short-circuits for normal users